pkg/jobrunaggregator/jobrunaggregatorlib/jobrun_locator.go

```diff
@@ -126,7 +130,12 @@ func (a *analysisJobAggregator) FindRelatedJobs(ctx context.Context) ([]jobrunag
 
 			if a.prowJobMatcher(prowJob) {
 				relatedJobRuns = append(relatedJobRuns, jobRunInfo)
-				break
+
+				// We run aggregated jobs 10 times, so once we find them all
```

Comment:
this is a configuration knob.  We cannot hardcode ten

Reply:
do you have a constant defined somewhere already? I can us it if so.